parse/fake_data_parse.go

```diff
@@ -89,22 +94,22 @@ func (fdp FakeDataParse) ReplaceVars(req *definition.Request, input string) stri
 func (fdp FakeDataParse) Parse(req *definition.Request, res *definition.Response) {
 	for header, values := range res.Headers {
 		for i, value := range values {
-			res.Headers[header][i] = fdp.ReplaceVars(req, value)
+			res.Headers[header][i] = fdp.ReplaceVars(req, res, value)
 		}
 
 	}
 	for cookie, value := range res.Cookies {
-		res.Cookies[cookie] = fdp.ReplaceVars(req, value)
+		res.Cookies[cookie] = fdp.ReplaceVars(req, res, value)
 	}
 
-	res.Body = fdp.ParseBody(req, res.Body, res.BodyAppend)
+	res.Body = fdp.ParseBody(req, res, res.Body, res.BodyAppend)
```

Comment:
Actually the thing is that the ParseBody method is used for getting AMQP body:
per.AMQP.Body = msender.Parser.ParseBody(req, res, per.AMQP.Body, per.AMQP.BodyAppend)
as long as getting response body:
res.Body = fdp.ParseBody(req, res, res.Body, res.BodyAppend)
, not found body:
res.Body = fbp.Parser.ParseBody(req, res, res.Persisted.NotFound.Body, res.Persisted.NotFound.BodyAppend)
and body loaded from file:
 res.Body = fbp.Parser.ParseBody(req, res, string(fileContent), res.Persisted.BodyAppend)